src/ImageSharp.Processing/Transforms/ResizeProcessor.cs

```diff
@@ -3,7 +3,7 @@
 // Licensed under the Apache License, Version 2.0.
 // </copyright>
 
-namespace ImageSharp.Processors
```

Comment:
done... I'm sure It had some logic when I moved them... can't remember what it was now 😕